src/partials/loader.html

```diff
@@ -0,0 +1,13 @@
+
+<!-- <body> -->
+    
+<figure class="loader" id="loader">
```

Comment:
a to sobie podlinkowujesz do każdego z plików html: indes, catalog i my-library jako partials zgadza się?